configure.ac

```diff
@@ -225,6 +225,36 @@ then
 fi
 AC_SUBST([ENABLED_ASM])
 
+# wolfEngine Options
+# This requires a FIPS source code bundle.
+AC_ARG_ENABLE([engine],
+    [AS_HELP_STRING([--enable-engine],[Enable wolfEngine options (default: disabled)])],
+    [ ENABLED_WOLFENGINE=$enableval ],
+    [ ENABLED_WOLFENGINE=no ]
+    )
+
+AS_CASE([$ENABLED_WOLFENGINE],
```

Comment:
This `AS_CASE` needs to move after the `AC_ARG_ENABLE` for FIPS below. Otherwise, the `ENABLED_FIPS` assignments here just get wiped out by that `AC_ARG_ENABLE`. Please test these changes with a FIPS bundle (that's how I uncovered this problem).

Reply:
Yes. This is on purpose. 
If you do 
```
--enable-engine --enable-fips=ready
```
You'll get fips-ready.  That's a good thing.  I wouldn't want `--enable-engine` 's fips setting to override `--enable-fips` settings.

Reply:
I see what you're saying, but if you just do `--enable-engine` and nothing else in a FIPS source tree, the configure will error out. The assignments to `ENABLED_FIPS` in your case statement have no effect.